docs/functions.rst

```diff
@@ -1607,6 +1607,7 @@ Arguments:
    `container`, Yes, `string`, name of the kanister sidecar container
    `includePath`, Yes, `string`, path of the data to be backed up
    `snapshotTags`, No, `string`, custom tags to be provided to the kopia snapshots
+   `userHostname`, No, `string`, user's hostname from user access map to be provided to the kopia repository server for performing backup
```

Comment:
We should change it to `ReposiotryServerUserHostname`

Reply:
Done !